cmd/tmpfs/tmpfs.go

```diff
@@ -1,3 +1,5 @@
+// +build go1.14
```

Comment:
Why not add this to the mod file?

Reply:
put up a PR, I don't know how to do that. Thanks.